proof/refine/AARCH64/CSpace_R.thy

```diff
@@ -229,7 +229,8 @@ lemma parent_preserved:
    isMDBParentOf cte' (CTE src_cap src_node)"
   using parency unfolding weak_derived'_def
   apply (cases cte')
-  apply (simp add: isMDBParentOf_CTE sameRegionAs_def2)
+thm isArchFrameCap_capMasterCap
```

Comment:
This `thm` is removed in a later commit but probably should get rid of it here to start with.

Reply:
- [ ] commit surgery